numpy/_core/meson.build

```
@@ -1051,7 +1051,7 @@ foreach gen_mtargets : [
     'loops_unary_complex.dispatch.h',
     src_file.process('src/umath/loops_unary_complex.dispatch.c.src'),
     [
-      AVX512F, [AVX2, FMA3], SSE2,
+      X86_V3, X86_V2,
```

```suggestion
      X86_V4, X86_V3, X86_V2,
```
AVX512 is missing here.